include/z64.h

```diff
@@ -80,7 +80,7 @@ typedef struct {
 // End of RDRAM with the Expansion Pak installed
 #define EXPANDED_RDRAM_END 0x80800000
 // Address at the end of normal RDRAM after which is room for a screen buffer
-#define FAULT_FB_ADDRESS (NORMAL_RDRAM_END - sizeof(u16[SCREEN_HEIGHT][SCREEN_WIDTH]))
+#define FAULT_FB_ADDRESS ((void*)(NORMAL_RDRAM_END - sizeof(u16[SCREEN_HEIGHT][SCREEN_WIDTH])))
```

Comment:
```suggestion
#define FAULT_FB_ADDRESS (void*)(NORMAL_RDRAM_END - sizeof(u16[SCREEN_HEIGHT][SCREEN_WIDTH]))
```
Very nitpicky, but don't feel the extra parens are needed.